### DISCUSSION TITLE
[Question] endReached offset

### DISCUSSION THREAD

**Discussion by igorpost92** [petyosi/react-virtuoso#387] [Q&A] [answered]
Hi.
As far as i understand, endReached callback fires when i scroll exactly to the end of the list.
Is it possible to fire this callback before the end with some offset, for example where there is 10 rows left to scroll or some distance in pixels?

**Comment by petyosi** [comment, marked as answer]
There's no offset configuration for that event. You can follow the `rangeChanged` event for that purpose.

**Reply by igorpost92** [reply]
Thank you, i'll try it
